lib/src/git.rs

```diff
@@ -323,21 +323,32 @@ pub fn export_refs(
     let view = mut_repo.view();
     // This list includes refs known to jj, namely all git-tracking refs and refs
     // for local jj branches.
-    // Short-term TODO: use the fact that git refs other than local branches are
-    // included.
     let jj_known_refs: HashSet<_> = view
         .git_refs()
         .keys()
         .filter_map(|name| parse_git_ref(name))
         .chain(
             view.branches()
                 .keys()
-                .map(|branch| RefName::LocalBranch(branch.to_owned())),
+                .map(|branch| RefName::LocalBranch(branch.to_owned()))
+            // Since remote-tracking branches are not listed here, this function is not able to
+            // export the creation of a remote-tracking branch. This is OK, as
+            // remote-tracking branches are only updated by `import_refs` after a fetch or a push.
+            // We *do* need to worry about exporting deletion of remote-tracking branches by `jj
+            // branch forget`.
```

Comment:
nit: that's probably true, but I prefer including remote branches if that doesn't introduce unexpected behavior. Fewer exceptions are better.

Anyway, we can change that later. I think this is good at this point.

Reply:
I was going back and forth on this. I'm OK with including them, especially if you are OK with it from the performance perspective. (I didn't think the performance perspective through, I leave all the optimization to you or others or a later version of me that learned to profile things).

I may change this PR accordingly, or we can change it afterwards as you said.

Reply:
> I'm OK with including them, especially if you are OK with it from the performance perspective.

I don't think that would matter. `RemoteBranch {..}` would have already been enqueued by `git_refs()` in most cases.